examples/IBFE/explicit/ex0/example.cpp

```diff
@@ -293,7 +293,7 @@ main(int argc, char* argv[])
         {
             for (unsigned int d = 0; d < NDIM; ++d)
             {
-                u_bc_coefs[d] = NULL;
+                u_bc_coefs[d] = nullptr;
```

Comment:
Could we move this modernization stuff to another PR?

Reply:
Sure. I got carried away. 😁

Reply:
I am going to split this into 3 commits --- one for the core library, one for the tests, one for the examples.